### DISCUSSION TITLE
Add Time Filter to Claim List

### DISCUSSION THREAD

**Discussion by SporyTike** [RetroAchievements/RAWeb#1112] [Ideas]
Since recently I use the Claim List to get data for the monthly Set of the Month voting and while it is easier to follow than a Google sheet or something similar it is a bit annoying to not be able to filter for a specific time frame for when the initial claim was made or when it was finished/dropped/expired. This leads me to manually dig through the list to find the beginning and ending of the entries I need to consider for the voting.
Also since everytime technically a new claim could be finished, everytime I move to the next page I need to double check that nothing was missed due to a new claim being finished and the next entry moving to the page I previously went through.

It would be nice to have a filter criteria for the initial claim date as well as a filter criteria for the Expiration/Completion/Drop Date so you can check claims from a specific time span without much extra work or being worried to miss something.

**Comment by StingX2** [comment]
Honestly I found it to be an unusable nightmare to figure out on both claims and releases. Ended up using the doc > copying to my private doc and running my auto format I whipped up. God bless Cobalt
